### ISSUE TITLE
Announcement Style is the same as Info style

### ISSUE THREAD

**Issue by jprovasnik** [learningworks/moodle-block_advnotifications#22]
There are 5 styles to choose from when setting up a notification: Info, Success, Warning, Danger, Announcement

When Announcement is selected, it is identical to Info (according to the CSS when viewing the browser tools). 

Is it possible to make "announcement" a separate class? I'd love to make it a nice purple on my site.

**Comment by zpottie** [comment]
Hey @jprovasnik 

Thanks for your submission! The plugin was made like that by design. It allows the admin of the site to create custom styling for the announcement type of alert in their theme, custom CSS/SCSS settings or additional HTML settings.

The CSS might look something like:
```CSS
.alert.alert-announcement {
    background-color: #a6c;
    color: #fff;
    border-color: #93c;
}
```

Hope that helps! 👌 

Thanks,
Zander

**Comment by jprovasnik** [comment]
Hey, I appreciate that, but the problem isn't that I cannot style it, it's that .alert-announcement doesn't seem to exist as a class for me to override. 
For example, when I select "Announcement" as the type, and then open my browser tools, it shows as .alert-info instead of .alert-announcement

<img width="222" alt="announcement" src="https://user-images.githubusercontent.com/44345287/77654725-cebbd980-6f47-11ea-8852-1c3f8882221c.png">
<img width="310" alt="class" src="https://user-images.githubusercontent.com/44345287/77654734-d0859d00-6f47-11ea-8760-e8e9f81e215a.png">

**Comment by zpottie** [comment]
Ah yes, that's just for the preview alert shown when setting it up. When you save it and it displays for real, it has an `announcement` class appended like so:
![Announcement](https://user-images.githubusercontent.com/9076608/77718846-c0a1b200-7048-11ea-9e83-42c6c5ff478b.png)

**Comment by jprovasnik** [comment]
BTW, this is the most amazing plugin ever! 

OK --  that worked for me - when I saved and viewed the **alert-announcement** class on the actual course page, I saw my custom 'purple' color. 

So the question is how would I get this specialize color to appear in the preview? When I tell teachers to select the purple-colored "announcement" option, they all were complaining that it wasn't there (since it turns out the preview wasn't showing a preview of the info-announcement class, it was instead showing a preview of the alert-info class). If that's not possible then no big deal - i can just tell them to save and then it's gonna be the purple color. 

Thanks : )

**Comment by zpottie** [comment]
I'm guessing when I initially thought up the preview alert idea, it was just to provide a rough idea of what it might look like. I'd recommend still following that philosophy 😅

I've pushed an update to the plugin, so from the latest commit it should add the announcement class to the preview alert - adfd052

You can download the latest master branch version [here](https://github.com/learningworks/moodle-block_advnotifications/archive/master.zip)

After you've updated the plugin, be sure to purge the caches on your site: `<webroot>/admin/purgecaches.php`

**NB** - The latest version of the plugin requires at least Moodle v3.5.3

**Comment by zpottie** [comment]
🎉 All fixed and official in the latest release - v1.3.8 🎉

Available from Moodle Plugins Directory now too.

Closing this...
